deposit/src/test/java/edu/unc/lib/deposit/normalize/AbstractNormalizationJobTest.java

```diff
@@ -70,10 +69,22 @@
 	protected String jobUUID;
 	protected String depositUUID;
 	protected File depositDir;
+	
+	private PID premisEventPid;
```

Comment:
Since this is only used in the initBase method, I would prefer if it were initialized in that scope